python312Packages.pycaption: 2.2.11 -> 2.2.12

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.pycaption is: Closed caption converter

meta.homepage for python312Packages.pycaption is: https://github.com/pbs/pycaption

meta.changelog for python312Packages.pycaption is: https://github.com/pbs/pycaption/blob/2.2.12/docs/changelog.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 2.2.12 with grep in /nix/store/mnyn7rwrzffc8wf232qyx440g5kgc0k4-python3.12-pycaption-2.2.12
- found 2.2.12 in filename of file in /nix/store/mnyn7rwrzffc8wf232qyx440g5kgc0k4-python3.12-pycaption-2.2.12

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

4 package rebuild(s)

First fifty rebuilds by attrpath

gnomecast
python311Packages.pycaption
python312Packages.pycaption
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/mnyn7rwrzffc8wf232qyx440g5kgc0k4-python3.12-pycaption-2.2.12 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.pycaption https://github.com/r-ryantm/nixpkgs/archive/371c9086981403fe4b719ef7887bb9ad389ba4f3.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/371c9086981403fe4b719ef7887bb9ad389ba4f3#python312Packages.pycaption
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/mnyn7rwrzffc8wf232qyx440g5kgc0k4-python3.12-pycaption-2.2.12
ls -la /nix/store/mnyn7rwrzffc8wf232qyx440g5kgc0k4-python3.12-pycaption-2.2.12/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>gnomecast</li>
    <li>gnomecast.dist</li>
    <li>python311Packages.pycaption</li>
    <li>python311Packages.pycaption.dist</li>
    <li>python312Packages.pycaption</li>
    <li>python312Packages.pycaption.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dotlambda for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda commented on pkgs/development/python-modules/pycaption/default.nix:
```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
+    hash = "sha256-vdSoqNWEubK0np14EiI/hRbrEx0722qg6Ggcpmhz8FM=";
   };
 
   nativeBuildInputs = [ setuptools ];
```
```suggestion
  build-system = [ setuptools ];
```

dotlambda committed:
python312Packages.pycaption: 2.2.11 -> 2.2.12

Diff: https://github.com/pbs/pycaption/compare/refs/tags/2.2.11...2.2.12

Changelog: https://github.com/pbs/pycaption/blob/2.2.12/docs/changelog.rst

dotlambda commented:
Result of `nixpkgs-review pr 338219` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>gnomecast</li>
    <li>gnomecast.dist</li>
    <li>python311Packages.pycaption</li>
    <li>python311Packages.pycaption.dist</li>
    <li>python312Packages.pycaption</li>
    <li>python312Packages.pycaption.dist</li>
  </ul>
</details>
